packages/preferences-persistence/src/create/debounce-async.js

```diff
@@ -9,23 +9,21 @@
  * This is distinct from `{ debounce } from @wordpress/compose` in that it
  * waits for promise resolution.
  *
- * @param {Function} func    A function that returns a promise.
- * @param {number}   delayMS A delay in milliseconds.
- *
- * @return {Function} A function that debounce whatever function is passed
- *                    to it.
+ * @return {Function} A function that debounces the async function passed to it
+ *                    in the first parameter by the time passed in the second
+ *                    options parameter.
  */
-export default function debounceAsync( func, delayMS ) {
+export default function createAsyncDebouncer() {
```

Comment:
nit: now the filename doesn't match the default export.

Reply:
Renamed 👍